php/class-fieldmanager-group.php

```diff
@@ -1,4 +1,9 @@
 <?php
+/**
+ * Class file for Fieldmanager_Group.
+ *
+ * @package Field / Group
```

Comment:
Not sure if this conversation was had at some point, but two things here:
1) According to the [PHPDoc.org docs](https://www.phpdoc.org/docs/latest/references/phpdoc/tags/package.html), `@package` should be the "logical subdivision" of the [structural element](https://www.phpdoc.org/docs/latest/glossary.html#term-structural-element), separated by `\` (similar to namespaces: `@package [level 1]\[level 2]\[etc.]`).
2) There shouldn't be [multiple `@package` tags](https://github.com/alleyinteractive/wordpress-fieldmanager/pull/591/files#diff-65cae30651b720e59f90adc96c1edd35R15) for one structural element.